src/components/Searchbar/Searchbar.js

```diff
@@ -1,6 +1,7 @@
 import React, { useState, useEffect } from "react";
 import ResultField from "./ResultField";
-import CloseButton from "./CloseButton";
+import InputField from "./InputField";
+import ClearInput from "./ClearInput";
 import atractions from "./dummyData"; // podmień na dumm data z projektu kiedy powstanie
```

Comment:
Small spelling mistake - it should be "attractions".

Reply:
ok :)

Reply:
`import React, { useState, useEffect } from "react";`

- useEffect is not used - Please remove it. 

`{searchInput.length !== 0 && (
        <>
          <ResultField
            atractionList={atractionList}
            searchInput={searchInput}
          />

          <ClearInput clearInput={clearInput} />
        </>
      )}`

- Spelling mistake -  should be attractionList not atractionList

- React.Fragment <></> can be removed

Reply:
ok, good points:)
React fragment needs to stay I think